docs/_includes/src-highlightjs.adoc

```diff
@@ -0,0 +1,22 @@
+////
+Included in:
+
+- user-manual: Source Code Syntax Highlighting: HighlightJs installation
+
+:highlightjs-lang: https://highlightjs.org/download/
+////
+
+{highlightjs-org}[HighlightJS] is a popular client-side syntax highlighter that supports a broad range of {highlightjs-lang}[languages].
+
+To use highlightjs just add `:source-highlighter: highlightjs` in your AsciiDoc file.
+
+[IMPORTANT]
```

Comment:
I don't think this needs to be an admonition block. Let's just go with normal content.